source/uwp/Renderer/lib/Util.cpp

```diff
@@ -92,6 +92,19 @@ std::string HStringToUTF8(winrt::hstring const& in)
     return WStringToString(in);
 }
 
+std::optional<double> TryHStringToDouble(winrt::hstring const& in)
+{
+    try
+    {
+        return std::stod(winrt::to_string(in));
```

Comment:
check if `_wtof` is available and may be use that instead?
you can use it like:
```suggestion
        return _wtof(in.c_str());
```